api/src/main/java/liquibase/ext/datatype/core/MySQLBooleanType.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package liquibase.ext.datatype.core;
+
+import liquibase.database.Database;
+import liquibase.database.core.MySQLDatabase;
+import liquibase.datatype.DatabaseDataType;
+import liquibase.datatype.core.BooleanType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
```

Comment:
How about putting some documentation here explaining why we need these liquibase classes?

Reply:
Yep, will do.

Reply:
Done.